src/Server_Command.php

```diff
@@ -1,4 +1,5 @@
 <?php
+use WP_CLI\Utils;
```

Comment:
Let's add a line break here, imports are a "section" for themselves.
```suggestion

use WP_CLI\Utils;
```